spec/game_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+require_relative '../lib/game.rb'
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+
+describe Game do
+    let(:player1) { 'Rohan' }
+    let(:player2) { 'Vivek' }
+    let(:marker1) { 'X' }
+    let(:marker2) { 'O' }
+    subject(:game) { Game.new(player1, player2) }
+
+    describe "Retrieve player names and markers" do
+        it "Check if the Game object was created" do
+            expect(game).to be_a Game
+        end
+
+        it "Retrieve player1's name. Should be Rohan" do
+            expect(game.player1.name).to eq(player1)
+        end
+
+        it "Retrieve player2's name. Should be Vivek" do
+            expect(game.player2.name).to eq(player2)
+        end
+
+        it "Retrieve player1's marker. Should be X" do
+            expect(game.player1.marker).to eq(marker1)
+        end
+
+        it "Retrieve player2's marker. Should be O" do
+            expect(game.player2.marker).to eq(marker2)
+        end
+
+        it "If new game is initialized the game state should be :active" do
```

Comment:
- [ ] Game Retrieve player names and markers If new game is initialized the game state should be :active